internal/inspect/utils.go

```diff
@@ -0,0 +1,68 @@
+package inspect
+
+import (
+	"embed"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+	"time"
+
+	"github.com/go-errors/errors"
+	"github.com/jackc/pgconn"
+	"github.com/jackc/pgx/v4"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/supabase/cli/internal/db/reset"
+	"github.com/supabase/cli/internal/utils"
+)
+
+const ROOT_DIR = "queries/"
+const CSV_QUERY = `COPY (%s) TO STDOUT WITH CSV HEADER`
+
+//go:embed queries/*.sql
+var queries embed.FS
+
+func ReadQuery(query string) string {
+	path := fmt.Sprintf("%s%s.sql", ROOT_DIR, query)
+	queryString, err := queries.ReadFile(path)
+	if err != nil {
+		println(err.Error())
+		return ""
+	}
+	return string(queryString)
+}
+
+func Report(cmd *cobra.Command, config pgconn.Config, fsys afero.Fs, options ...func(*pgx.ConnConfig)) error {
+	queries, err := queries.ReadDir("queries")
+	if err != nil {
+		return err
+	}
+	date := time.Now().Format("2006-01-02")
+	out, err := cmd.Flags().GetString("out-path")
```

Comment:
Instead of supporting a flag, can we print everything to stdout and let users redirect to a file?

We can be more flexible with output format this way, as users can choose json / yaml / csv / etc.

Reply:
I would advise against this because special chars can cause users to need to run `reset` and larger outputs would go beyond the scrollback history. 
If they want to do something with the output, then they could run each command in the current behaviour

Reply:
Ok, we can keep this flag since it writes to multiple files. Shall we rename it as `output-dir` so it's clearer?